microprofile/WFLY-17138-convert-mp-metrics-subsystem-to-legacy-subsystem.adoc

```diff
@@ -0,0 +1,75 @@
+= WFLY-17138 - Convert MP Metrics Subsystem to Legacy Mode
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Red Hat has publicly stated our disagreement with the direction of MicroProfile Metrics and feel Micrometer, a de facto industry standard, provides a better metrics solution for our customers. To that end, Micrometer support will be added (via another JIRA) and, under this JIRA, MP Metrics support will be removed from standard configurations and module moved to "legacy" mode.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-17138[WFLY-17138]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-17144[WFLY-17144]
+* https://issues.redhat.com/browse/EAP7-1686[EAP7-1686]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [X] Engineering
+
+* [X] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+// Remove the x next to the relevant field if the feature in question is not relevant
+// to that kind of WildFly installation
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [x] Managed domain
+
+* [x] OpenShift s2i
+
+* [x] Bootable jar
+
+== Requirements
+
+=== Hard Requirements
+
+* MicroProfile Metrics susbstem will be removed from standard configs
+
+=== Nice-to-Have Requirements
+
+=== Non-Requirements
+
+== Backwards Compatibility
+
+Since this represents the removal of a subsystem, any deployment dependent on the APIs provided will no longer deploy or un.
```

Comment:
The current analysis template has a number of subsections that should be covered here.

Reply:
@bstansberry, I've added the missing sections and relevant content. Please verify that these changes are sufficient.